dart/lib/src/protocol/sentry_user.dart

```diff
@@ -64,33 +74,64 @@ class SentryUser {
   ///
   /// These keys are stored as extra information but not specifically processed
   /// by Sentry.
-  final Map<String, dynamic>? extras;
+  final Map<String, dynamic>? data;
+
+  @Deprecated('Will be removed in v7. Use [data] instead')
+  Map<String, dynamic>? get extras => data;
```

Comment:
If I set any `extras` to the user, and capture an event, does the `extras` field appear in the UI? I wonder if we should fall back to `data` or just deprecate and ignore it.
I don't see `extras` mapped in the [protocol](https://develop.sentry.dev/sdk/event-payloads/types/#user).

Reply:
Yep, `extras` is shown in the UI as follows for 
```dart
scope.setUser(SentryUser(
  email: 'foo@bar.org',
  extra: {'foo': 'bar'},
));
```

<img width="377" alt="Bildschirmfoto 2022-10-03 um 15 41 49" src="https://user-images.githubusercontent.com/1270149/193592448-c17c7aa9-48c4-428c-bbb1-a4424ecddf4c.png">

`data` is shown as follows for
```dart
scope.setUser(SentryUser(
  email: 'foo@bar.org',
  data: {'foo': 'bar'},
));
```

<img width="413" alt="Bildschirmfoto 2022-10-03 um 15 41 23" src="https://user-images.githubusercontent.com/1270149/193592484-0a473405-e81f-4b46-b080-ec1808b4c1bf.png">

Reply:
Ok, so `extras` are not dropped during ingestion, thanks.